doc/ssl/SSL_SESSION_get_time.pod

```diff
@@ -52,6 +59,8 @@ valid values.
 
 SSL_SESSION_set_time() and SSL_SESSION_set_timeout() return 1 on success.
 
+SSL_SESSION_get0_hostname returns the current value.
```

Comment:
I think it is about the hostname as defined in https://www.ietf.org/rfc/rfc3546.txt. And I believe it will never fail, but possibly be NULL.